karma.conf.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function(config) {
+  config.set({
+    browsers: ['PhantomJS'],
+    frameworks: ['jasmine'],
+    files: [
+      'edx_notifications/static/js/HelloWorld.js',
+      'edx_notifications/static/js/specs/hello-world-spec.js'
+    ],
```

Comment:
might there be a way to inspect the files on disk to build up this array of 'files' so that as we build out more Backbone JS files as well as test specs, we don't need to add them here?

Is karma.conf.js a NodeJS executable? If so, it seems like we should be able to access the filesystem.